java/java.lsp.server/src/org/netbeans/modules/java/lsp/server/protocol/TextDocumentServiceImpl.java

```diff
@@ -2243,4 +2285,220 @@ protected CallHierarchyOutgoingCall createResultItem(CallHierarchyItem item, Lis
         };
         return t.processRequest();
     }
+
```

Comment:
This is not against this PR, but this file is really long :(

Reply:
Agree :-(